tests/test_dsp.py

```diff
@@ -93,6 +93,34 @@ def test_group_delay_single_channel(impulse_group_delay):
         grp, impulse_group_delay[1].flatten(), rtol=1e-10, atol=1e-10)
 
 
+def test_complex_group_delay_single_channel(impulse_complex_group_delay):
+    """Test the function returning the group delay of a signal,
+    single channel."""
+    signal = impulse_complex_group_delay[0]
+
+    with pytest.raises(ValueError, match="Invalid method"):
+        dsp.group_delay(signal, method='invalid')
+
+    with pytest.raises(ValueError, match="not supported"):
+        dsp.group_delay(signal, method='fft', frequencies=[1, 2, 3])
+
+    grp = dsp.group_delay(signal, method='scipy')
+    assert grp.shape == (signal.n_bins, )
+    npt.assert_allclose(
+        grp, impulse_complex_group_delay[1].flatten(), rtol=1e-10, atol=1e-10)
+
+    grp = dsp.group_delay(signal, method='fft')
+    assert grp.shape == (signal.n_bins, )
+    npt.assert_allclose(
+        grp, impulse_complex_group_delay[1].flatten(), rtol=1e-10, atol=1e-10)
+
+    grp = dsp.group_delay(
```

Comment:
This looks like the same test case as above. If true, it could be deleted

Reply:
Yes, but here it's tested for the 'fft' method. The test above is for 'scipy' method. I think its good to test, that both methods do the same for complex signals.

Reply:
The comment might have moved, I was referring to the code below, where there are two test for `grp = dsp.group_delay(signal, method='fft')`